src/rdbms-connect/azext_rdbms_connect/custom.py

```diff
@@ -0,0 +1,111 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=import-error,too-many-locals,too-many-statements
+import os
+from knack.util import CLIError
+from knack.log import get_logger
+import psycopg2
+import pymysql
+
+logger = get_logger(__name__)
+DEFAULT_MYSQL_DB_NAME = 'flexibleserverdb'
+DEFAULT_PG_DB_NAME = 'postgres'
+
+
+def connect_to_flexible_server_mysql(cmd, server_name, administrator_login, administrator_login_password=None,
+                                     database_name=None, querytext=None, interactive_mode=False):
+    mysql_server_endpoint = cmd.cli_ctx.cloud.suffixes.mysql_server_endpoint
+    return connect_to_server_helper("mysql", mysql_server_endpoint, DEFAULT_MYSQL_DB_NAME, server_name,
+                                    administrator_login, administrator_login_password, database_name,
+                                    querytext, interactive_mode)
+
+
+def connect_to_flexible_server_postgres(cmd, server_name, administrator_login, administrator_login_password=None,
+                                        database_name=None, querytext=None, interactive_mode=False):
+    postgresql_server_endpoint = cmd.cli_ctx.cloud.suffixes.postgresql_server_endpoint
+    return connect_to_server_helper("postgres", postgresql_server_endpoint, DEFAULT_PG_DB_NAME, server_name,
```

Comment:
avoid positional arguments

Reply:
Assuming this is only in reference to interactive_mode? Is it ok to use None for the rest of the params? I will modify interactive_mode to None as well if that is ok.